src/resolve.js

```diff
@@ -58,7 +58,7 @@ const createResolve = (config) => {
       const hashmap = getHashmapFromComment(content)
 
       if (hashmap) {
-        cache[filename] = hashmap
+        cache[filename] = { [hashmap.name]: { [config.name]: hashmap.id } }
```

Comment:
`cache[filename]` is kind of `{ComponentA: {'data-id': 'hash'}, ComponentB: {'data-id': 'hash'}, ...}`
so you need to iterate through the `hashmap` fields and map values to something like `{config.name}: data.id`